src/style/_markdown.scss

```diff
@@ -0,0 +1,47 @@
+@import 'node_modules/@hmcts/ccd-case-ui-toolkit/dist/cut-styles/components/typography';
```

Comment:
don't think we should me manging scc like this... it should be in import from node module... 
if u do decide to leave it please to do and speak to mo and andy for a solution